sdk/communication/azure-communication-rooms/README.md

```diff
@@ -25,7 +25,7 @@ add the direct dependency to your project as follows.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-rooms</artifactId>
-  <version>1.0.0-beta.1</version>
+  <version>1.0.0-beta.2</version>
 </dependency>
 ```
 
```

Comment:
After line 30 you should have the following end tag
`[//]: # ({x-version-update-end})`
This tells the tooling to stop scanning, that the version has been updated.

Reply:
Let me fix this.

Reply:
Added the end tag. Thanks Jim.